app/models/user.rb

```diff
@@ -9,6 +9,7 @@ class User < ApplicationRecord
 
   validates :email, presence: true, uniqueness: true, format: { with: URI::MailTo::EMAIL_REGEXP }
   validate :ensure_valid_profile_image
+  validates :default_period, inclusion: { in: Period::PERIODS.keys }, allow_nil: true
```

Comment:
To match what's happening at the DB-level, we should remove the `allow_nil: true` from this validation